src/redux/actions/userActions.js

```diff
@@ -272,25 +272,25 @@ export const getUserPromos = (id) => (dispatch) => {
   dispatch({ type: types.GET_USER_FRIENDS_START, payload: true });
   API.get(`savedPromos/users/${id}`)
     .then((res) => {
-      console.log(res.data);
       dispatch({
         type: types.GET_USER_PROMOS,
         payload: res.data,
       });
     })
     .catch((err) => {
```

Comment:
I noticed in writing these tests that these .catch blocks are handling errors through the console instead of using an action-creator. We should look at this before our final day to make sure the user can see something on the DOM if an error occurs.